Assets/BossRoom/Scripts/Server/DebugCheatsManager.cs

```diff
@@ -0,0 +1,67 @@
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+namespace Unity.Multiplayer.Samples.BossRoom.Server
+{
+    /// <summary>
+    /// Handles debug cheat events, applies them on the server and logs them on all clients. This class is only
+    /// available in the editor or for development builds
+    /// </summary>
+    public class DebugCheatsManager : NetworkBehaviour
+    {
+        [SerializeField]
+        [Tooltip("Enemy to spawn. Make sure this is included in the NetworkManager's list of prefabs!")]
+        NetworkObject m_EnemyPrefab;
+
+        [SerializeField]
+        [Tooltip("Boss to spawn. Make sure this is included in the NetworkManager's list of prefabs!")]
+        NetworkObject m_BossPrefab;
+
+        public void SpawnEnemy()
+        {
+            SpawnEnemyServerRpc();
+        }
+
+        public void SpawnBoss()
+        {
+            SpawnBossServerRpc();
+        }
+
+        public void GoToPostGame()
+        {
+            GoToPostGameServerRpc();
+        }
+
+        [ServerRpc(RequireOwnership = false)]
+        void SpawnEnemyServerRpc(ServerRpcParams serverRpcParams = default)
```

Comment:
Since your call sites for these RPCs don't use serverRpcParams, I believe you can just remove "default"

Reply:
They do use it, for the SenderClientId. This is used for the "snitch log", but it is required anyways for the god mode cheat, if we want to apply it to only a single player.

Reply:
aaand that's what happens when you read too fast.
nvm